### PR TITLE
mbs.jp

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [x] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

https://www.mbs.jp/jounetsu/
https://www.mbs.jp/monthly/

### Add your comment and screenshots

<details><summary>screenshot</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SocialFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -40,6 +40,7 @@ thepythoncode.com#?#div[align="center"]:has(> span:contains(/^Sharing is caring/
 verstka.media#?#main em + a[href="https://t.me/svobodnieslova"]:upward(1)
 verstka.media#?#main div[class^="wp-container-"]:has(> h2.verstka-telegram-link:not([style]))
 rlx.jp#?#.whiteBg > div.sectionBox:has(> div.inner > div.shearBox)
+mbs.jp#?#.foot-sns > div[class^="foot-sns_inner"]:upward(1)
```
Comment (Yuki2718) [inline SocialFilter/sections/css_extended.txt:43]: Why not `##.foot-sns`? False positive on other page?

Comment (SKEIDs) [reply SocialFilter/sections/css_extended.txt:43]: I thought `##.foot-sns` is a general rule to may cause problems on other webpages.
Do I worry too much about it?

Comment (Yuki2718) [reply SocialFilter/sections/css_extended.txt:43]: I mean `mbs.jp##.foot-sns`

Comment (SKEIDs) [reply SocialFilter/sections/css_extended.txt:43]: yeah, I'm also talking about it.

**Review by Yuki2718 — APPROVED** [review]

**Review by Sergey-Lyapin — APPROVED** [review]

**Pull request merged**